source/content/guides/account-mgmt/workspace-sites-teams/07-teams.md

```diff
@@ -262,6 +280,24 @@ Workspace Administrators, Users in Charge, or Site Owners can add a [Supporting
 
    ![Site with two Supporting Workspaces](../../../../images/dashboard/manage-site-team.png)
 
+## Remove a Supporting Workspace from a Site
+
+1. [Go to the Site Dashboard](/guides/account-mgmt/workspace-sites-teams/sites#site-dashboard) with the Supporting Workspace you wish to remove.
+
+1. Click **<span class="glyphicons glyphicons-group"></span> Team**.
+
+1. Click the **x** to the right of the Supporting Workspace you wish to remove.
+
+1. If you are a Diamond or Platinum contract customer, you will be provided additional options to remove the user's access from associated sites and will need to select one of the following:
```

Comment:
@rachelwhitton I saw you just push the word `contract`. But my understanding was that this functionality is available to customers on any contract. Not just Diamond or Platinum contracts.

Reply:
cc @zeroxiii

Reply:
Slack discussion internally indicates it's available to contract workspaces with Diamond and Platinum support tiers only

Reply:
My mistake, fixed in most recent commit